app.go

```diff
@@ -240,6 +246,31 @@ func executeQuote(cmd command) bool {
 		return false
 	}
 
+	//Check auto buy sell
+
+	for _, v := range (*autoBuyRequestStore)[stock] {
+		//fmt.Printf("key[%s] value[%s]\n", k, v)
+		if v.Trigger.ToFloat() < quote.Price.ToFloat() {
+			//Fulfil buy action
+			wholeShares, cashRemainder := quote.Price.FitsInto(v.Amount)
```

Comment:
if `v.Amount == $0.00`, then `cashRemainder == quote.Price`. We should `break` from this case if `wholeshares == 0`

https://github.com/DistributedDesigns/currency/blob/1317dabc0bbdfea9da9dda8ea8124ad09db799b0/currency_test.go#L97